packages/api/amplify_api/test/amplify_api_config_test.dart

```diff
@@ -0,0 +1,84 @@
+import 'dart:convert';
+
+import 'package:amplify_api/src/amplify_api_config.dart';
+import 'package:amplify_core/amplify_core.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'test_data/fake_amplify_configuration.dart';
+
+void main() {
+  late EndpointConfig endpointConfig;
+
+  group('GraphQL Config', () {
+    const endpointType = EndpointType.graphQL;
+    const endpoint =
+        'https=//abc123.appsync-api.us-east-1.amazonaws.com/graphql';
+    const region = 'us-east-1';
+    const authorizationType = APIAuthorizationType.apiKey;
+    const apiKey = 'abc-123';
+
+    setUpAll(() async {
+      const config = AWSApiConfig(
+          endpointType: endpointType,
+          endpoint: endpoint,
+          region: region,
+          authorizationType: authorizationType,
+          apiKey: apiKey);
+
+      endpointConfig = const EndpointConfig('GraphQL', config);
+    });
+
+    test('should return valid URI with null params', () async {
+      final uri = endpointConfig.getUri(path: null, queryParameters: null);
+      final expected = Uri.parse('$endpoint/');
+
+      expect(uri, equals(expected));
+    });
+
+    test('should returns invalid URI with params', () async {
```

Comment:
question: What is this testing and why is it invalid? Seems like this logic is tested below (although for REST) so not sure this test is necessary. If you do keep it, nit on grammar of description "should return ...."

Reply:
Think you're right, this test isn't needed. It's not invalid per se, and it's not a unique test case.